### PR TITLE
chore: fix release build

### PR DESCRIPTION
**Pull request by busunkim96**
Fix release build by migrating to secret manager secrets and use templated kokoro configs for docs/ and release/

### CODE REVIEW

File: .kokoro/trampoline.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright 2020 Google LLC.
+# Copyright 2017 Google Inc.
```
Comment (vi3k6i5) [inline .kokoro/trampoline.sh:2]: nit: please update this.

File: .kokoro/populate-secrets.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Copyright 2020 Google LLC.
```
Comment (vi3k6i5) [inline .kokoro/populate-secrets.sh:2]: nit: please update the year.

**Review by vi3k6i5 — APPROVED** [review]
LGTM to me, please fix the nits before submitting the changes.

**Review by vi3k6i5 — COMMENTED** [review]
Most of the rest of the code is BSD style license : https://github.com/googleapis/python-spanner-django/search?q=BSD-style can you make it consistent with that? There is some Apache license used here and there, we are looking to fix that.

**Review by busunkim96 — COMMENTED** [review]
License has been tweaked to use BSD. Some (trampoline.sh) have older years, as the files were originally authored in https://github.com/googleapis/synthtool/tree/master/synthtool/gcp/templates/python_library/.kokoro. When I last checked with OSPO I was told that code is copied into other repos should retain its original copyright year.

**Comment by vi3k6i5** [comment]
@busunkim96 : Can you fix the `Kokoro docs-presubmit — Build failed` issue, We can submit the change once it's fixed.

**Review by parthea — APPROVED** [review]

File: .kokoro/docker/docs/Dockerfile [hunk 3]
```diff
@@ -1,16 +1,8 @@
 # Copyright 2020 Google LLC
```
Comment (busunkim96) [inline .kokoro/docker/docs/Dockerfile:1]: @vi3k6i5 This should fix the keyserver error (the change in https://github.com/googleapis/synthtool/pull/1155). 

Sorry about sitting on this PR, I completely forgot about it 😅.

Owlbot should open PRs automatically if any of these files change at the template source (synthtool).

**Comment by busunkim96** [comment]
`docs-presubmit` is now green. @vi3k6i5 Please look over the files and merge. Thanks!

File: .kokoro/docker/docs/Dockerfile (hunk 3 above)
Comment (vi3k6i5) [reply .kokoro/docker/docs/Dockerfile:1]: No worries, Thanks for fixing the issues :)

**Pull request merged**